docs/docs/how-to/build-shards.md

```diff
@@ -0,0 +1,291 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# Build Shards
+
+!!! note
+    This guide is for new users unfamiliar with the Shards building process. 
+    
+    Click [here](#overview) to skip the tutorial and jump to the overview.
+
+How-to build Shards.
```

Comment:
The "How-to" + Page Name was used as the opening line previously so I have left it as such. I think it can be omitted since it is a little redundant. Will do so for the next commit.